README.md

```diff
@@ -30,3 +30,119 @@ Once the above variables are set:
 hack/run-ci-e2e-test.sh
 ```
 
+## Bundling the Windows Machine Config Operator
+This directory contains resources related to installing the WMCO onto a cluster using OLM.
+
+### Pre-requisites
+[operator-sdk](https://github.com/operator-framework/operator-sdk) and
+[opm](https://github.com/operator-framework/operator-registry/) have been installed on the localhost.
+
+### Generating a new bundle
+If changes need to be made to the bundle spec, a new bundle can be generated with:
+```shell script
+operator-sdk generate csv --csv-channel alpha --csv-version $NEW_VERSION --default-channel --operator-name windows-machine-config-operator --update-crds --from-version $PREV_VERSION
+```
+
+You should replace `$NEW_VERSION` and `$PREV_VERSION` with the new semver and the previous semver respectively.
+This will create a new directory: `deploy/olm-catalog/windows-machine-config-operator/$NEW_VERSION`
+You should use this new directory when [creating the bundle image](#creating-a-bundle-image)
+
+### Creating a bundle image
+A bundle image can be created by editing the CSV in deploy/bundle/windows-machine-config-operator/manifests/
+and replacing `REPLACE_IMAGE` with the location of the WMCO operator image you wish to deploy.
+See [the build instructions](#build) for more information on building the image.
+
+You can then run the following command in the root of this git repository
+```shell script
+operator-sdk bundle create $BUNDLE_REPOSITORY:$VERSION_TAG -d deploy/bundle/windows-machine-config-operator/manifests \
```

Comment:
This command needs to updated to look like operator-sdk bundle create quay.io/ssoto/test:bundlev0.0.0 -d deploy/olm-catalog/windows-machine-config-operator/0.0.0 --channels alpha --default-channel alpha --image-builder podman
will do that after reviews